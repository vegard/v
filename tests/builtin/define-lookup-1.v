# crash, should be error
t := fun u64 ();
t () u64 0;
