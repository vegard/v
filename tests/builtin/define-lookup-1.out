tests/builtin/define-lookup-1.v:3:0: cannot access value at compile time
t () u64 0;
^
